### ISSUE TITLE
Deriving disease modules from the compressed transcriptional space embedded in a deep autoencoder

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#5]
## TL;DR
This paper hypothesize that disease modules - from molecular interaction maps, could be discovered within deep neural network 'autoencoder' representations.

### Paper Link

https://www.nature.com/articles/s41467-020-14666-6

### Author/Institution

Lefteris Koumakis (Foundation for Research and Technology - Hellas)

## Methods

Paper aims at using deep neural networks for identification of a new unbiased data-driven functional representation that can explain complex diseases without the reliance of the PPI network which is known to be incomplete and strongly affected by the study bias of some early discovered cancer genes.

## Cite

Dwivedi, S.K., Tjärnberg, A., Tegnér, J. et al. Deriving disease modules from the compressed transcriptional space embedded in a deep autoencoder. Nat Commun 11, 856 (2020). https://doi.org/10.1038/s41467-020-14666-6

## Comments

Microarray and RNA-seq transcriptomics data, autoencoder deep neural network.

Review paper
